var/spack/repos/builtin/packages/py-psyclone/package.py

```diff
@@ -7,36 +7,69 @@
 
 
 class PyPsyclone(PythonPackage):
-    """Code generation for the PSyKAl framework from the GungHo project,
-    as used by the LFRic model at the UK Met Office."""
+    """
+    Code generation system developed to support domain-specific languages
+    (DSLs) for finite element, finite volume and finite difference codes.
+    Notably, it is used in the LFRic Project from the UK Met Office, and
+    it also supports the GOcean (2D, finite difference) DSL.
 
+    """
+
+    # Links
     homepage = "https://github.com/stfc/PSyclone"
-    url = "https://github.com/stfc/PSyclone/archive/1.5.1.tar.gz"
     git = "https://github.com/stfc/PSyclone.git"
+    pypi = "PSyclone/PSyclone-2.5.0.tar.gz"
+
+    # License
+    license("BSD-3-Clause")
 
+    # Releases
     version("develop", branch="master")
-    version("1.5.1", commit="eba7a097175b02f75dec70616cf267b7b3170d78")
```

Comment:
Can you also add 1.5.1? We don't normally remove versions without first deprecating them.